jwql/tests/test_utils.py

```diff
@@ -32,6 +32,7 @@
     {'activity': '02',
```

Comment:
I apologize in advance of what's coming:
pylint report for this file: `Your code has been rated at -9.31/10`, mostly because of identation.
(pycodestyle also complains about it).
I leave it up to you to fix or not, I do not mind leaving it as is.

Reply:
Gah! PyCharm is great for a lot of things, but its linter lets me get away with a lot. I'll try to clean it up a bit.